### PR TITLE
Changes to work on render8

### PR DESCRIPTION
**Pull request by juaresba**
Applies the modifications so it works with render 8 as well.

### CODE REVIEW

File: template/vtex/manifest.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "vendor": "CHANGE_ME",
```
Comment (emersonlaurentino) [inline template/vtex/manifest.json:2]: you can change the `CHANGE_ME` of this file by the variables that were defined there in the package.json

like this https://github.com/vtex/create-react-library/blob/master/template/vtex/package.json#L2

File: template/vtex/react/Adapter.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    title: 'editor.CHANGE_ME.title',
+    description: 'editor.CHANGE_ME.description',
+    type: 'object',
+    properties: {},
+  },
+  text: "Default Text"
+}
+
+
+export default Adapter
```
Comment (emersonlaurentino) [inline template/vtex/react/Adapter.tsx:26]: Your eslint is configured?

File: template/vtex/store/interfaces.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "CHANGE_ME": {
```
Comment (emersonlaurentino) [inline template/vtex/store/interfaces.json]: use variable

**Review by emersonlaurentino — COMMENTED** [review]
the file `template/vtex/.vtexignore` is necessary?

File: template/vtex/manifest.json (hunk 1 above)
Comment (juaresba) [reply template/vtex/manifest.json:2]: done

File: template/vtex/store/interfaces.json (hunk 3 above)
Comment (juaresba) [reply template/vtex/store/interfaces.json]: done

File: template/vtex/react/Adapter.tsx (hunk 2 above)
Comment (juaresba) [reply template/vtex/react/Adapter.tsx:26]: I'm actually having some trouble with linting, but I think this time I got it right

**Comment by klzns** [comment]
@emersonlaurentino @juaresba Is something missing here? Can we merge this?

**Pull request merged**